frontends/ol-widgets/src/index.ts

```diff
@@ -1,2 +1,14 @@
+/**
+ * A TS project consuming this TS source code will try to typecheck this
+ * package. Since CKEditor doesn't have great typing, it will fail unless it has
+ * the ambients declarations in ./types/ckeditor.d.ts (or declares its own
+ * CKEditor types).
+ *
+ * Eslint wants this to be an import, but there is no type-only version of a
+ * sideffect import. So use old /// ts syntax. See https://github.com/microsoft/TypeScript/issues/36812
+ */
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
+/// <reference path="./types/ckeditor.d.ts" />
```

Comment:
This is annoying, and maybe the one downside of not compiling TS to JS for each package. But still very much worth it overall.

_Also, CKEditor types are getting **much** better. They have re-written a lot of the core functionality in TS, so we're starting to get builtin typings for it anyway._